### ISSUE TITLE
Training loss doesn't converge

### ISSUE THREAD

**Issue by bareblackfoot** [smallcorgi/Faster-RCNN_TF#68]
I tried to train the model you provided, but it didn't converge.
I didn't change any code.
Is there anyone who succeed to train?

Thank you

**Comment by bareblackfoot** [comment]
After training, I tested the trained weights and got the test results as the readme.md file says. 
But I think It's because there is a good initialization values in the pretrained model. The training of the faster-RCNN is not working for some reason in TensorFlow. I guess it could be the gradient problem caused by no gradient operations like tf.cast. 
I'm not sure, but I will try to find and if I find it, I will post it here.

**Comment by wwxJream** [comment]
@bareblackfoot I also meet this problem. I haven't change any code. When I train this model, the loss becomes nan. I need to change the learning rate to 0.00001. But I find that the loss can't stable. Do you have any idea?

**Comment by indestinee** [comment]
@bareblackfoot I have the same problem, My box-loss is always 0 when training and I got nothing when testing, have you fixed it?

**Comment by bareblackfoot** [comment]
@wwxJream @indestinee I decided to use caffe version! It solves the problem and we can also debug several files there which are implemented as python layer here.

**Comment by indestinee** [comment]
@bareblackfoot thank you, I will try caffe too.
